Containers/UserProfile/UserProfile.js

```diff
@@ -13,9 +13,23 @@ export default class UserProfile extends Component {
         }
     }
     render() {
+        const { profilePic } = this.props;
         return (
-            <View style={styles.container}>
-                <Icon color="#3c4859" name="portrait" type="badge" size={150} />
+            <View style={styles.viewContainer}>
+                {
+                    profilePic.length ?
+                        <TouchableOpacity style={styles.profilePictureContainer} onPress={() => this.props.takeProfilePic()}>
+                            <View style={styles.pictureContainer} >
```

Comment:
The Image component wouldn't take a css property of display: block, so I wrapped the image in a view so it could be centered.